proto/minder/v1/minder.proto

```diff
@@ -2915,7 +2999,7 @@ message ListEvaluationHistoryRequest {
         (buf.validate.field).repeated = {
             items: {
                 string: {
-                    pattern: "[,[:word:]]*",
+                    pattern: "^[,[:word:]]*$",
```

Comment:
Why do we allow "," in a repeated field?

Reply:
Because of - https://github.com/mindersec/minder/blob/3872633d436f63fda7c3f0cb842258153946f6bb/internal/controlplane/handlers_evalstatus.go#L283-L287